src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -1525,9 +1532,32 @@ public function registerGlobalScopes($builder)
      */
     public function newQueryWithoutScopes()
     {
-        return $this->newModelQuery()
-            ->with($this->with)
-            ->withCount($this->withCount);
+        return $this->withAggregates(
+            $this->newModelQuery()
+                ->with($this->with)
+                ->withCount($this->withCount)
+        );
+    }
+
+    /**
+     * Add default model aggregates to a query builder.
+     *
+     * @param  \Illuminate\Database\Eloquent\Builder|static  $query
+     * @return \Illuminate\Database\Eloquent\Builder|static
+     */
+    protected function withAggregates(Builder $query)
```

Comment:
Note:  I don't love this method, or how it's called in newQueryWithoutScopes.  Open for improvement.